tests/test_tensor/test_model.py

```diff
@@ -213,9 +220,11 @@ def run_1d_hybrid_tp(model_name):
             assert torch.allclose(loss.torch_tensor(), loss_torch, rtol=1e-2)
 
         loss.backward()
+        colo_optimizer.step()
 
         if rank == 0:
             loss_torch.backward()
+            colo_optimizer_torch.step()
```

Comment:
check grads and parameters after each step

Reply:
Done